Explorer/Assets/DCL/Input/Crosshair/CrosshairCanvas.cs

```diff
@@ -0,0 +1,76 @@
+using UnityEngine;
+using UnityEngine.UIElements;
+using Utility.UIToolkit;
+
+namespace DCL.Input.Crosshair
+{
+    public class CrosshairCanvas : VisualElement, ICrosshairView
+    {
+        private VisualElement crossHairElement;
+        private Sprite crossHair;
+        private Sprite crossHairInteractable;
+        private Length leftLength;
+        private Length bottomLength;
+        private CursorStyle currentState;
+        private bool initialized;
+        private VisualElement cursorElement;
+
+        public void Initialize(Sprite crosshair, Sprite crosshairInteractable)
+        {
+            crossHairInteractable = crosshairInteractable;
+            crossHair = crosshair;
+            if (initialized) return;
```

Comment:
Just a curiosity, do we have an initialization phase that can be called multiple times? Can't we split init and crosshair and crosshairInteractable assignation in 2 separate parts?

Reply:
I believe this additionality safety is redundant - calling `Initialize` is fully under our direct control